src/vs/workbench/contrib/webview/browser/pre/index.html

```diff
@@ -112,6 +112,10 @@
 				color: var(--vscode-textLink-foreground);
 			}
 
+			p > a {
+				text-decoration: var(--text-link-decoration);
+			}
+
```

Comment:
@mjbvz do you know why updating this is causing the walkthrough markdown to not render?

![CleanShot 2024-06-21 at 16 08 26@2x](https://github.com/user-attachments/assets/4d5f0201-12a4-4561-ba4f-3f3e0dcf1360)

Reply:
After changing anything in this file, you also need to update the `content-security-policy` `sha` at the top. You can copy the one from the error message and update the line that the top that looks like:

```
	content="default-src 'none'; script-src 'sha256-EJBlYt9sOvo6RhZZJG+duTqIZCB8pctk1a62h/ZIElc='
```

Reply:
Got it, thanks!